### DISCUSSION TITLE
Oauth signup/login doesn't work for me

### DISCUSSION THREAD

**Discussion by Mahmouddd21** [open-webui/open-webui#4940] [Q&A]
I have open-webui on my laptop and was trying to include the signup and login. The button to continue with Google appears in both forms and when I click it, it tells me to pick the account and I allow access to login with certain scopes. After that immediately in the logs it shows an unknown session ID (Image attached). 
![image](https://github.com/user-attachments/assets/e64e650b-06de-4d39-a35e-d872735c04fd)
after that, it shows that there are no errors and that the connection was successful, but after it redirects to the login page, and when I check the admin page dashboard, it doesn't show any new users.

Can anyone who has worked with Oauth and made it work help me on this issue?

**Comment by lyz-code** [comment]
What worked for me was to run the log in process until it wen't through :S
